custom/covid/management/commands/update_case_index_relationship.py

```diff
@@ -15,11 +15,12 @@
 DEVICE_ID = __name__ + ".update_case_index_relationship"
 
 
-def should_skip(case, traveler_location_id):
+def should_skip(case, case_type, traveler_location_id):
+    if case_type == 'contact' and case.get_case_property('has_index_case') == 'no':
```

Comment:
The more aggressively you can simplify conditionals, the better - this function is clearer now that the triple-condition checks are gone. It looks like you could pull out the `len(case.indices) != 1` into a single `if len(case.indices) != 1: return True` statement before the `traveler_location_id is None` check, which would remove a bit more repetition.